src/satellite/test.js

```diff
@@ -270,7 +275,7 @@ describe('Satellite()', () => {
       const testRoute = async () => {
         const res = await fetch(`${url}/router/sub-router`);
         expect(res.ok).toBe(true);
-        service.stop(done);
```

Comment:
I removed this since `afterEach()` already does `service.stop()`

Reply:
I see now, unfortunate, we can't showcase mocks using the same client. 
I'll be fixing up some ES mock tests when upgrading ES to 8.0 anyways, I'll try to get tests to the way I want then.